src/Conversion/ONNXToKrnl/Math/Reduction.cpp

```diff
@@ -664,26 +625,20 @@ struct ONNXReduceSumOpLowering : public ConversionPattern {
         divisor = rewriter.create<arith::IndexCastOp>(
```

Comment:
There is a cast that gets index to float

Reply:
done